Library/Homebrew/cmd/uses.rb

```diff
@@ -77,24 +82,37 @@ def uses
 
   def intersection_of_dependents(use_runtime_dependents, used_formulae, args:)
     recursive = args.recursive?
+    only = if args.formula?
+      :formula
+    elsif args.cask?
+      :cask
+    end
     includes, ignores = args_includes_ignores(args)
 
+    deps = []
     if use_runtime_dependents
-      used_formulae.map(&:runtime_installed_formula_dependents)
-                   .reduce(&:&)
-                   .select(&:any_version_installed?) +
-        select_used_dependents(
+      unless only == :cask
+        deps += used_formulae.map(&:runtime_installed_formula_dependents)
+                             .reduce(&:&)
+                             .select(&:any_version_installed?)
+      end
+      unless only == :formula
```

Comment:
```suggestion
      elsif only != :formula
```
but might be easier to read still if these are refactored to be `only == :cask`

Reply:
This can't be `elsif` because both need to be checked.